packages/showcases/svelte/stories/components/navigation/VtmnSearch.stories.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf';
+  import { VtmnSearch } from '@vtmn/svelte';
+</script>
+
+<Meta
+  title="Components / Navigation / VtmnSearch"
+  component={VtmnSearch}
+  argTypes={{
+    variant: {
+      description: 'Color variant of search component.',
+      defaultValue: 'default',
+    },
+    disabled: {
```

Comment:
disabled: {
    type: { name: 'boolean', required: false },
    description: 'The disabled state of the input.',
    defaultValue: false,
    control: { type: 'boolean' },
  },